dashboard/config/programming_classes/javalab/scene.json

```diff
@@ -1,244 +1,265 @@
 {
-  "category_key": null,
-  "content": null,
-  "examples": null,
-  "external_documentation": null,
-  "fields": null,
+  "category_key": "org_code_theater",
+  "content": "",
+  "examples": "[]",
+  "external_documentation": "",
+  "fields": "[]",
   "key": "Scene",
   "name": "Scene",
-  "syntax": null,
-  "tips": null,
+  "syntax": "",
+  "tips": "",
   "methods": [
     {
-      "key": "getwidth",
-      "position": 0,
       "name": "getWidth()",
+      "position": 0,
```

Comment:
I'm assuming you didn't do anything that would affect this order. I'll take an action item to fix this

Reply:
no I just changed the category. Let me know if I should manually change it back, I can do that.

Reply:
nah I quickly looked over it and it looks good so I'd keep it as is. We've had issues in the past with these serializations having noisy diffs so I'll fix that in the next few days